frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/networking-tab/vm-wizard-nic-modal-enhanced.tsx

```diff
@@ -60,6 +60,7 @@ const VMWizardNICModal: React.FC<VMWizardNICModalProps> = (props) => {
       allowPodNetwork={allowPodNetwork}
       nic={networkInterfaceWrapper}
       network={networkWrapper}
+      submitButtonText='Add'
```

Comment:
can we use the constant here from strings file?

Reply:
Done!